tests/languages/rescript/char_feature.test

```diff
@@ -0,0 +1,19 @@
+'b'
+'\''
+'\\'
+'\xff'
+'\o214'
+
+----------------------------------------------------
+
+[
+	["char", "'b'"],
+	["char", "'\\'"], ["operator", "'"],
+	["operator", "'"], "\\\\", ["operator", "'"],
```

Comment:
Well, seems like my simple pattern was a little too simple. Could we replace the char regex with `'(?:[^\r\n\\]|\\(?:.|\w+))'`?

Reply:
Yeah